iree/base/internal/file_io_test.cc

```diff
@@ -27,82 +23,36 @@ namespace {
 
 using ::iree::testing::status::StatusIs;
 
-std::string GetUniquePath(absl::string_view unique_name) {
+std::string GetUniquePath(const char* unique_name) {
   char* test_tmpdir = getenv("TEST_TMPDIR");
+  if (!test_tmpdir) {
+    test_tmpdir = getenv("TMPDIR");
+  }
+  if (!test_tmpdir) {
+    test_tmpdir = getenv("TEMP");
+  }
   IREE_CHECK(test_tmpdir) << "TEST_TMPDIR not defined";
```

Comment:
This comment is no longer accurate

Reply:
Why? The _test_ still checks for `TEST_TMPDIR` first.

Reply:
fixed